### PR TITLE
Add Contribute command

### PR DESCRIPTION
**Pull request by ooTruffle**
This tag tells users how they are able to add a new tag to Janator if they think something should be added

### CODE REVIEW

File: src/commands/index.ts [hunk 1]
```diff
@@ -4,5 +4,6 @@ import english from "./tags/english"
 import basestation from "./tags/basestation"
 import support from "./tags/support"
 import getserverinfo from "./tags/getserverinfo"
+import contribute from "./tage/contribute"
```
Comment (thewilloftheshadow) [inline src/commands/index.ts]: Typo

**Review by thewilloftheshadow — CHANGES_REQUESTED** [review]

**Review by ooTruffle — COMMENTED** [review]
i... i can english i swear

**Review by thewilloftheshadow — APPROVED** [review]

**Comment by ooTruffle** [comment]
Also added the "app" change

**Review by SinOfStrife — APPROVED** [review]
Lgtm

**Pull request merged**